### ISSUE TITLE
bbox loss always zero

### ISSUE THREAD

**Issue by Root-9527** [google/automl#45]
1. Why bbox loss always zero(show in tensorboard) while training on signle GPU; anchor and gt mach problem?
2. always problem as followed
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'tuple' object has no attribute 'name'
ERROR:tensorflow:Model diverged with loss = NaN.
E0325 20:20:13.613487 140640882104064 basic_session_run_hooks.py:768] Model diverged with loss = NaN.
WARNING:tensorflow:Reraising captured error
W0325 20:20:14.657608 140640882104064 error_handling.py:142] Reraising captured error

**Comment by TomHeaven** [comment]
Probably you made a mistake in converting the COCO dataset to tf_records. Refer to
 > 
@Alex220284, I believe you've managed to solve your training problem and closed the issue - that's great. Would you be kind enough to share your solution?

I have made a mistake on creating the training tfrecord by setting the param "caption_annotations_file" to "captions_train2017.json". In fact, Since I rectify the command to set param "object_annotations_file" to "instances_train2017.json", all things go as expected. Hope this is helpful to you.

_Originally posted by @Alex220284 in https://github.com/google/automl/issues/91#issuecomment-605915288_

**Comment by Root-9527** [comment]
> Probably you made a mistake in converting the COCO dataset to tf_records. Refer to
> 
> > 
> 
> @Alex220284, I believe you've managed to solve your training problem and closed the issue - that's great. Would you be kind enough to share your solution?
> 
> I have made a mistake on creating the training tfrecord by setting the param "caption_annotations_file" to "captions_train2017.json". In fact, Since I rectify the command to set param "object_annotations_file" to "instances_train2017.json", all things go as expected. Hope this is helpful to you.
> 
> _Originally posted by @Alex220284 in [#91 (comment)](https://github.com/google/automl/issues/91#issuecomment-605915288)_
thanks, i have fixed it

**Comment by mingxingtan** [comment]
Hi @Root-9527 Could you paste your command? Or try out our tutorial first?  Since it runs well on colab tutorial, I am closing this issue for now, but feel free to reopen it if you still have problems. Thanks!
